src/qcodes_contrib_drivers/drivers/Tektronix/FCA3100.py

```diff
@@ -3,7 +3,7 @@
 from typing import Any, Tuple
 from qcodes.instrument import VisaInstrument
 from qcodes.parameters import MultiParameter, Parameter, ParameterWithSetpoints
-from qcodes.validators import Arrays, Ints
+from qcodes.utils.validators import Arrays, Ints
```

Comment:
This changes the import from the current official location to a deprecated legacy location. Could you change this back